performance_optimization_stack/infra/airbyte/main.tf

```diff
@@ -0,0 +1,53 @@
+// Airbyte Terraform provider documentation: https://registry.terraform.io/providers/airbytehq/airbyte/latest/docs
+
+
+// Sources
+resource "airbyte_source_datadog" "my_source_datadog" {
```

Comment:
@utsavbhattarai007 I didn't find "airbyte_source_datadog" in https://registry.terraform.io/providers/airbytehq/airbyte/latest/docs 

Maybe the docs are not up to date. Have you tested this configuration to check if it actually works?